### ISSUE TITLE
build a database of known junk code strings

### ISSUE THREAD

**Issue by williballenthin** [mandiant/flare-floss#773]
if our code recovery solution (lancelot or vivisect) fails to identify some code, then we may still display some junk strings that are actually instructions, like 

```
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ .text ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┫┃
It<Iu4P                                                             000099c3                  ┃┃
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ .rdata ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┫┃
```

its likely that this particular instruction sequence is not difficult to recover generally; rather, in this one sample the code analyzer lost trace of a function. therefore, if we recovered code ranges on a large number of programs and matched that up with the strings extracted from the same files, we could build a database of strings that are likely instruction sequences. we could use this database as a fallback to further filter junk strings after the code recovery pass.

**Comment by williballenthin** [comment]
interesting. do you mean browsing strings from windows binaries across versions? or something else?

**Comment by Vulcanraven91** [comment]
Yes, you could then also look up when the string occurred for the first time.

**Comment by mr-tz** [comment]
Attached are two JSONL files containing strings
- from the `.text` sections of
- thousands of C:\Windows native binaries
- occurring 100 times or more

While not perfect, it's an easy approximation (with some obvious FPs).

The files are split into strings of length
1. 4-5 characters and
2. 6 or more characters

This is only due to an extraction approach I took earlier.

[text_section_strings.zip](https://github.com/mandiant/flare-floss/files/11666592/text_section_strings.zip)

**Comment by mr-tz** [comment]
Above and below are using 480ca51ba24be6f3ad72ce5282b28783. Below with min_len = 6.
Using a wider set (VT etc.) of samples hopefully provides better results.

Before
<img width="1903" alt="2023-06-07_15-34-23_WindowsTerminal" src="https://github.com/mandiant/flare-floss/assets/17606537/aa47b99b-5279-492b-8477-b8c7f3d4a9e9">

After
<img width="1897" alt="2023-06-07_15-32-01_WindowsTerminal" src="https://github.com/mandiant/flare-floss/assets/17606537/623dc7e9-fc72-42a5-9070-917afc4c7aba">
